tsdown.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'tsdown';
+
+export default defineConfig({
+  entry: [
+    'src/index.ts',
+    'src/base.ts',
+    'src/vitest/index.ts',
+    'src/vitest/base.ts',
+    'src/extensions.ts',
+  ],
+  format: ['cjs'],
```

Comment:
Chose to keep the bundle as CJS as I'm not certain that going to an ESM-only eslint config won't cause any issues. I don't have reason to believe it _will_ cause any issues, but I don't want to take any chances.

Maybe we can ship a CJS/ESM bundle in the near future.